### ISSUE TITLE
Zooming in causes zooming in and out repeatedly

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/molecules-and-light#209]
**Test Device**

Bates

**Operating System**

Windows 7

**Browser**

Internet Explorer

**Problem Description**

For https://github.com/phetsims/QA/issues/419.

If you try to use `ctrl` + `+` to zoom in, the sim will zoom in and out repeatedly on its own. (See gif below.) Clicking around will usually fix this.

**Visuals**

![issue](https://user-images.githubusercontent.com/31285590/64736365-b30b4d00-d4a7-11e9-974c-286530d65ed4.gif)

<details>
<summary><b>Troubleshooting Information</b></summary>
!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-customZoom.7/phet/molecules-and-light_en_phet.html?a11y
Version: 1.5.0-customZoom.7 2019-08-28 23:16:46 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.132 Safari/537.36
Language: en-US
Window: 1440x821
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by jessegreenberg** [comment]
Interesting, thanks @lmulhall-phet. Is it *only* from keyboard or do other methods like the trackpad or mouse  produce this too?

**Comment by lmulhall-phet** [comment]
Forgot to mention these two methods for zooming. Apologies.

Unfortunately, this device is old enough that its trackpad doesn't have the pinch-to-zoom-in/out feature, so I don't have a way to test this.

The mouse will also produce the behavior seen in the gif.

**Comment by jessegreenberg** [comment]
Not able to see this on my desktop with IE11 but I think I saw this on my laptop. So this is probably related to performance.

My guess is that we are overshooting destination scale when zooming in because dt is so large.

**Comment by jessegreenberg** [comment]
Not seeing this on my laptop either.

**Comment by jessegreenberg** [comment]
I see this just a little bit with window.phet.chipper.makeRandomSlowness

**Comment by jessegreenberg** [comment]
Ill bet we have the same problem with panning too.

**Comment by jessegreenberg** [comment]
Should be fixed, I am not seeing this in with window.phet.chipper.makeEverythingSlow().

However, when this happens I expect that performance is still pretty poor. If it is specific to Win 7 IE11 then it may not b etoo concerning.

**Comment by KatieWoe** [comment]
Looks good

**Comment by jessegreenberg** [comment]
Great, tahnk you
